pkg/virt-controller/watch/vm.go

```diff
@@ -2538,10 +2538,10 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 		vmCopy := vm.DeepCopy()
 
 		if c.clusterConfig.HotplugNetworkInterfacesEnabled() {
-			if err := c.handleInterfaceRequests(vmCopy, vmi); err != nil {
-				log.Log.Object(vm).Errorf("error encountered while handling network interface hotplug request: %v", err)
+			if err := c.handleDynamicInterfaceRequests(vmCopy, vmi); err != nil {
+				log.Log.Object(vm).Errorf("error encountered while handling dynamic network request: %v", err)
```

Comment:
nit: I think you used dynamic interface (vs network) in most other places, including method names.